Docs/development/jasp-build-guide-linux.md

```diff
@@ -24,7 +24,7 @@ Based on your system, you can install the mentioned libraries using your package
 On Ubuntu, you can use `apt`. 
 
 ```
-sudo apt install libboost libjsoncpp-dev libarchive-dev autoconf zlib1g zlib1g-dev cmake gfortran build-essential r-base
+sudo apt install libboost libjsoncpp25 libarchive13 autoconf zlib1g zlib1g-dev cmake gfortran build-essential r-base
```

Comment:
But if we dont have the development package versions we can't compile anymore right?

In any case @JohnnyDoorn also tried building on ubuntu this week and he should have a list of required packages perhaps?

DOn the other hand, if you @shun2wang  can build with this selection that could be ok, but to test it Id first have to arrange an Ubuntu install somewhere.

Reply:
I used this line that you gave me:
`sudo apt-get -y install build-essential ninja-build libxcb-xkb-dev libxcb-xkb1 libxkbcommon-dev libxkbcommon-x11-dev jags libreadstat-dev libjsoncpp-dev libboost-all-dev mercurial gfortran-9 gfortran-10 flex bison cmake libarchive-dev zlib1g-dev libzstd-dev mesa-utils libglu1-mesa-dev freeglut3-dev mesa-common-dev libglew-dev libglfw3-dev libglm-dev libao-dev libmpg123-dev`

Reply:
and these were especially helpful I think `libxkbcommon-dev libxkbcommon-x11-dev`

Reply:
This is minimum dependencies list here, there is no harm in using both shared and development libraries. as 
@JohnnyDoorn  pointed out, I did fighting with this  `libxkbcommon-dev libxkbcommon-x11-dev` too, Some of these libraries are installed from Conan(such` bison flex`) so we can add the necessary.

Reply:
I double checked and can now using these dependencies to compile on a fresh Ubuntu 22.04 system (although some dependency versions need to be upgraded separately, this can be seen in cmake)